Source/PowerMeter/PowerMeter.cs

```diff
@@ -43,7 +43,7 @@ public struct CommonConfiguration
             /// <summary>
             /// Specifies whether auto-averaging mode is enabled or not.
             /// </summary>
-            public bool Averaging;
+            public bool Averaging_Auto_Enabled;
```

Comment:
Jintao, for consistency with all other variable names in the RDLs I'd recommend this be `AveragingAutoEnabled`

Reply:
I see. Just renamed it. Please review again.